config.hpp.in

```diff
@@ -80,4 +93,13 @@
 
 #endif
 
+// temporary measure for MSVC versioning
+// language support is set to latest
+// at least MSVC 16.4.0 preview 2.0
+#if defined (_MSC_VER) && (_MSVC_LANG > 201703L) && (_MSC_FULL_VER > 192428206)
```

Comment:
Is it necessary to have the condition ``_MSVC_LANG > 201703L`` here? Or are there remaining issues when using the latest MSVC in C++17 mode?

Reply:
Unfortunately it is still necessary. If one removes it i.e. one disables the MSVC workarounds for the latest preview version of the compiler when C++17 is used quite a number of errors pop up. For me it looks like problems with lambda expressions used in "inline constexpr" expressions.